index.html

```diff
@@ -145,9 +145,30 @@ <h3>Skills</h3>
         </div>
       </article>   
     </section>
+    <section id="contact-form">
+      <h2 class="dark-blue playfair">Get Started</h2>
+      <p class="dark-blue poppins">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <form action="https://formspree.io/f/xpzegyyg" method="POST">
+        <label class="poppins" for="user-name">Your name</label>
+        <input type="text" id="user-name" name="user_name" placeholder="Anastasiia" maxlength="30" required>
+        <label class="poppins" for="user-email">Your email</label>
+        <input type="email" id="user-email" name="user_email" placeholder="anastasiia@gm..." required>
+        <label class="poppins" for="user-message">Your message</label>
```

Comment:
- [ ] [OPTIONAL] Please I suggest replacing the placeholder text with professional text. 😄 

```suggestion
        <input type="text" id="user-name" name="user_name" placeholder="Enter your name" maxlength="30" required>
        <label class="poppins" for="user-email">Your email</label>
        <input type="email" id="user-email" name="user_email" placeholder="Enter your email" required>
        <label class="poppins" for="user-message">Your message</label>
```